client/settings/fraud-protection/tour/index.tsx

```diff
@@ -0,0 +1,118 @@
+/**
+ * External dependencies
+ */
+import React, { useEffect, useState } from 'react';
+import { useDispatch } from '@wordpress/data';
+import interpolateComponents from 'interpolate-components';
+import { __ } from '@wordpress/i18n';
+
+/**
+ * Internal dependencies
+ */
+import Tour from '../../../components/tour';
+import { useSettings } from '../../../data';
+import tourStepImage from '../../../../assets/images/fraud-protection/tour/enhanced-protection-step.png';
+import tourStepImage2x from '../../../../assets/images/fraud-protection/tour/enhanced-protection-step@2x.png';
+
+const FraudProtectionTour: React.FC = () => {
+	const { isWelcomeTourDismissed } = wcpaySettings.fraudProtection;
+
+	const { isLoading } = useSettings();
+	const { updateOptions } = useDispatch( 'wc/admin/options' );
+	const [ showTour, setShowTour ] = useState( false );
+
+	useEffect( () => {
+		if ( ! isLoading && ! isWelcomeTourDismissed ) {
+			setShowTour( true );
+		}
+	}, [ isLoading, isWelcomeTourDismissed ] );
+
+	const handleTourEnd = () => {
+		updateOptions( {
+			wcpay_fraud_protection_welcome_tour_dismissed: true,
+		} );
+		wcpaySettings.fraudProtection.isWelcomeTourDismissed = true;
+
+		setShowTour( false );
+	};
+
+	if ( ! showTour ) return null;
+
+	return (
+		<Tour onTourEnd={ handleTourEnd }>
+			<Tour.Step
+				selector="#wpcontent"
+				position={ { bottom: 20, left: 20 } }
+			>
+				<Tour.Image
+					srcSet={ `${ tourStepImage }, ${ tourStepImage2x } 2x` }
+					src={ tourStepImage }
+					alt={ __( 'Enhanced fraud protection is here 🔒' ) }
+				/>
+
+				<Tour.Content
+					title={ __( 'Enhanced fraud protection is here 🔒' ) }
+					description={ __(
+						"Incoming transactions will now be screened for common risk factors, at the level of your choosing. Review any transactions caught by these filters and select whether you'd like to approve or decline them"
```

Comment:
This is most likely a design thing, but I feel like these sentences do not read well.

Commas in English mean to pause while reading, or they can split up compound sentences. In this instance, I don't feel like a pause is needed, and it is not a compound sentence.

> Incoming transactions will now be screened for common risk factors, at the level of your choosing.

For the second sentence, "these filters," are mentioned, but filters are not mentioned in the first sentence.

> Review any transactions caught by these filters and select whether you'd like to approve or decline them

A suggestion to replace both sentences would be:

> You can now choose a level of protection to have for screening incoming transactions. You will then be able to review any caught transactions and select whether you would like to approve or decline them.

Reply:
Thanks for your suggestion @jessepearson!

@elizaan36 what do you think about this change?

Reply:
As @elizaan36 is AFK, I'll proceed with the change. It was addressed on [6745aca](https://github.com/Automattic/woocommerce-payments/pull/5716/commits/6745aca1388460629b11952641aa93bcd301f7d4). Thanks again for your suggestion @jessepearson!

Reply:
Regarding the deprecation logs, I didn't find a better way to do it, and there are some other occurrences of the same function to update the options.

I'll create an issue to document this behavior. Thanks for catching it!